services/pg-backup/docker-compose.public.yml.j2

```diff
@@ -0,0 +1,5 @@
+version: '3.7'
+services:
+  {{ PREFIX_STACK_NAME }}_pg_backup:
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Why does this need to have the docker socket mounted for public?

Reply:
It is from the precedent pg-backup. I will remove it. Thanks